lib/validator.js

```diff
@@ -0,0 +1,20 @@
+const { getCaster } = require('./types');
```

Comment:
This validator.js file should start with a capital letter because it's a class. Convention is to capitalize class files. 
should be Validator.js but don't make this change because github won't recognize the update.
 this is more a side note.